test1.html

```diff
@@ -39,15 +39,15 @@ <h1>SPECIALIZED CLINIC</h1>
   </header>
   <main class="page-main">
     <section class="breadcrumbs">
-      <a href="#"><img src="#banner"></a>
-      <ul>
-        <li>Home/</li>
-        <li>Health Service</li>
+      <a href="#"><img src="#banner" alt="banner"></a>
+      <ul class="breadcrumb-list">
+        <li class="breadcrumb-item">Home/</li>
+        <li class="breadcrumb-item">Health Service</li>
       </ul>
     </section>
     <aside class="sidebar-left">
       <article class="category">
-        <h4>ADULTS</h4>
```

Comment:
Should not type upper-case this text.
Using CSS for transform the text to upper-case